db-tests/src/test/java/TestTriggers.java

```diff
@@ -1067,5 +1068,105 @@ SELECT COUNT(*)
         }
       }
     }
+
+    @Test
+    void shouldCancelOnMissionModelUpdate() throws SQLException {
+      try (final var statement = connection.createStatement()) {
+        statement
+            .executeUpdate(
+                """
+                    UPDATE mission_model
+                    SET name = 'updated-name-%s'
+                    WHERE id = %s;"""
+                    .formatted(UUID.randomUUID().toString(), missionModelId)
```

Comment:
What was the motivation to generate a UUID rather than just using a hardcoded string for this test?

Reply:
Theres a uniqueness constraint across mission name and version that I am avoiding here.